blog/_posts/2018-11-03-Team101DaysOfSummer-at-OSSEU.md

```diff
@@ -0,0 +1,187 @@
+
+![Hall](/img/blog/2018/OSSEU5.jpg)
+<div class="image-credits">Source: Paavini’s camera</div>
+
+
+It was almost lunch time by the time we ended up visiting the booths, so we went to attend the Women in Open Source lunch by Adobe at Sheraton Grand. There we met so many other awesome women, some of who were already in their full time jobs and were a way more experienced in open source than us. 
+
+
+![Sheraton-lunch](/img/blog/2018/OSSEU6.jpg)
+<div class="image-credits">Source: Paavini’s camera</div>
+
+Next, we attended a session on Machine Translation by - Suneel Marthi. He explained how he and his team at Amazon tried building a German to English language translator and discussed the intricacies of the problem.
+
+
+![Talk](/img/blog/2018/OSSEU7.jpg)
+<div class="image-credits">Source: Paavini’s camera</div>
+
+The talk broadly discussed about Statistical(SMT) and Neural(NMT) methods to solve this problem. Statistical methods included making alignment functions on the basis of word positions, phrase based conversions etc. In context to Neural methods, attention based mechanism and byte-pair encoding was discussed. It was interesting to note that the latter was a paper by the University of Edinburgh who had won several competitions using this.
+There was also a brief talk on Sockeye, Flink and Amazon Sagemaker which were the platforms tried and tested by Suneel. 
+
+Next in line was the session on Scalable Machine Learning with Kubeflow by Barbara Fusinska.
+
+![Talk](/img/blog/2018/OSSEU8.png)
+<div class="image-credits">Source: Paavini’s camera</div>
+
+
+The major aspects covered in this talk were the questions, what is machine learning, deep learning and artificial intelligence? It felt like a discovery to know that Kubeflow was actually the amalgamation of Kubernetes + Tensorflow! We discussed how data scientist’s viewed any problem. The architecture for any problem had three major components: algorithm choosing, model building and model choosing plus serving. Different technology stacks which can be used for these three parts was further taken into consideration. It was a light talk made interactive with QA session and fun quotes such as the one shown above! 
+P.S. The quote on the shirt of the scientist above said “I invented the satellite but all I got was this lousy t-shirt” :)
+ 
+The day ended with the All-Attendee Reception held at the National Museum of Scotland. It was a beautiful place with people dressed in traditional clothes and bagpiper music filling the air. We also got a glimpse of the statue of Dolly, the first cloned sheep.The live performance by the energetic performers further cheered up everyone. We met people from working on linux kernel and media streaming and went back happily to our warm and cozy hotels.
+
+![Live Performance](https://www.instagram.com/stories/highlights/17917470982245763/)
```

Comment:
Hi @paavininanda ! Above instagram link doesn't seem to exist. Can you please check and revise the post accordingly? Blog post looks good otherwise. Thanks

Reply:
@nerdyvaishali Hi, yes it works only if the person is logged into Instagram. Anyway I have updates the post accordingly. Thanks 👍